### ISSUE TITLE
[Alert, Share & Post] cannot add ne messeges

### ISSUE THREAD

**Issue by jordan8396** [LSS-Manager/lss-manager-v3#793]
There is a bug with the Alert, Share & Post add now, where I cannot add a new -pre-programmed message. There is an "+" sign and what appears to be an add button. but it does not work.
I also had an issue where I accidentally deleted all the messages (not knowing what to do) and now I cannot add any new ones - I tried to disable then re-enable the addon but it doesn't change anything.

**Comment by KBOE2** [comment]
Could be a localStorage Problem.
You need to clear the localStorage entry for shareAlliancePost:
1. Press F12
2. Go to "application" or "web store" (depends on browser)
3. Click "localStorage"
4. Right click the LSSM_SHARE_ALLIANCE_POST entry (could have similar name) and click delete
5. Reload game.

Does that help?

**Comment by jordan8396** [comment]
Hi, I cleared ended up clearing my browser's cache, which resolved the issue of deleting the default messages (i can now edit them again). I can also add new messages now too.
However, I've just checked and if you delete all current messages, the issue occurs again and you cannot add a new one.

**Comment by KBOE2** [comment]
ok, thanks for your report. I will investigate this.
Just for clarification: You deleted each message itself and then you're unable to create new messages?

**Comment by jordan8396** [comment]
Yes, I deleted each message with the "-" and once there were no messages left I was unable to create a new one.

**Comment by SaibotH90** [comment]
I had the same issue. Clearing the local storage helped. However, the button to restore the default settings did not help, nor did the add button.
